src/breadcrumbs/styled-components.js

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+
+// @flow
+
+import {styled} from '../styles/index';
+import type {StyledRootPropsT, StyledSeparatorT} from './types';
+import {ChevronRight} from '../icon';
+
+export const StyledRoot = styled('nav', ({$theme}: StyledRootPropsT) => {
+  return {
+    color: $theme.colors.breadcrumbsText,
+    ...$theme.typography.font450,
+  };
+});
+
+export const StyledSeparator = styled('div', ({$theme}: StyledSeparatorT) => {
+  return {
+    display: 'inline-block',
+    color: $theme.colors.breadcrumbsSeparatorFill,
+    marginLeft: $theme.sizing.scale300,
+    marginRight: $theme.sizing.scale300,
+  };
+});
+
+export const StyledIcon = styled(ChevronRight, () => {
+  return {
+    verticalAlign: 'text-bottom',
```

Comment:
it only works for inline parent element. If Separator's styles or tag is changed this won't work. How about to use flexbox model to style it?

Reply:
With the `flex` model it will center the icon relative to the container; not the text. This makes the icon look slightly off-center in some instances.  

Want to go over some possible styling offline and find a better match?